tools/XPlatInstall/install_ch.sh

```diff
@@ -61,12 +60,13 @@ GET_LATEST()
     CHAKRA_VERSION=$(curl -k -s -SL "https://aka.ms/chakracore/version")
 
     if [[ "$CHAKRA_VERSION" == *"Error"* ]]; then
-        PRINT $ERROR_COLOR "] Download Failed. Do you have an Internet connection?"
+        PRINT $ERROR_COLOR "] Download Failed. Do you have Internet connection?"
         exit 1
     fi
 
     PRINT $SUCCESS_COLOR "] Found ChakraCore ${CHAKRA_VERSION//_/.} for ${CURRENT_OS}"
-    BINARY_NAME="https://aka.ms/chakracore/${BINARY_NAME}_${CHAKRA_VERSION}.tar.gz"
+    BINARY_NAME="https://aka.ms/chakracore/${BINARY_NAME}_${CHAKRA_VERSION}"
```

Comment:
Okay, got it. So it's still a .tar.gz file downloaded from a url without dots, and the script saves it with the correct extension.